extensions/jackson/runtime/src/main/java/io/quarkus/jackson/ObjectMapperProducer.java

```diff
@@ -17,9 +21,20 @@
     @Produces
     public ObjectMapper objectMapper(Instance<ObjectMapperCustomizer> customizers) {
         ObjectMapper objectMapper = new ObjectMapper();
-        for (ObjectMapperCustomizer customizer : customizers) {
+        List<ObjectMapperCustomizer> sortedCustomizers = sortCustomizersInDescendingPriorityOrder(customizers);
+        for (ObjectMapperCustomizer customizer : sortedCustomizers) {
             customizer.customize(objectMapper);
         }
         return objectMapper;
     }
+
+    private List<ObjectMapperCustomizer> sortCustomizersInDescendingPriorityOrder(
```

Comment:
That looks odd, is inverting the priority also done somewhere else?

Reply:
Higher priority means that the module should take precedence in case of the same module has been applied in a lower priority customizer. 
Same as where higher priority in MP Config means that the Config Source take precedence